l10n_ec_ote/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "OTE for Ecuador",
+    "summary": "OTE for Ecuador",
+    "category": "Localization",
+    "author": "Odoo Community Association (OCA)",
```

Comment:
Shouldn´t you add yourself as author too?

Reply:
@simahawk I add CONTRIBUTORS and Odoo-EC as Author, is OK?